doc/dev/developer_guide/basic-workflow.rst

```diff
@@ -295,6 +295,38 @@ to change the `Issue tracker`_ status to "Resolved". Some issues may be
 flagged for backporting, in which case the status should be changed to
 "Pending Backport" (see the `Backporting`_ chapter for details).
 
+See also `Merging`_ for more information on merging.
+
+Proper Merge Commit Format
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This is the most basic form of a merge commit::
+
+       doc/rados: remove upgrade auth disable
+
+       Reviewed-by: Reviewer Name <rname@example.com>
+
+This consists of two parts:
+
+#. The title of the commits of the pull request to be merged.
+#. The name and email address of the reviewer. Enclose the reviewer's email 
+   address in angle brackets.
+
+If you cannot find the email address of the reviewer on his or her github
+page, you can look it up in the **.githubmap** file, which can be found in
+the repository at **/ceph/.githubmap**.
```

Comment:
Sometimes `.githubmap` lookup is not enough. [This method](https://www.sourcecon.com/how-to-find-almost-any-github-users-email-address/) could find almost any github user's email address. I think you should mention this method also here.

Reply:
> Sometimes `.githubmap` lookup is not enough. [This method](https://www.sourcecon.com/how-to-find-almost-any-github-users-email-address/) could find almost any github user's email address. I think you should mention this method also here.

@joscollin, do you think that that page is durable so that I can just link to it, or do you think that it'd be better if I wrote a version of that page here in the Ceph docs?

Reply:
> wrote a version of that page here in the Ceph docs

The later is better. 

.githubmap is not always updated and not every reviewer has an entry in `.githubmap`. ptl-tool has flags that [turn off .githubmap updates](https://github.com/ceph/ceph/pull/36124) too.

Reply:
i think the right solution is to fix `.githubmap` instead of asking developer to fire up a browser for search reviewer's email.